### PR TITLE
Adjust language switcher GUI

### PR DESCRIPTION
**Pull request by pablothedude**
The GUI of the language switcher should be optimized in order to be
more user friendly.

Also prevented submitting the form manually after the selection is changed.

### CODE REVIEW

**Comment by pablothedude** [comment]
https://www.pivotaltracker.com/n/projects/1163646/stories/165635661

File: src/Surfnet/StepupRa/RaBundle/Resources/public/js/stepup-ra.js [hunk 1]
```diff
@@ -188,5 +188,11 @@
             $('button#ra_search_ra_second_factors_export').hide();
         }
 
+        // Post sub institution switcher on change
+        $(document).on('change', '#select_institution_institution', function() {
```
Comment (MKodde) [inline src/Surfnet/StepupRa/RaBundle/Resources/public/js/stepup-ra.js]: Got to love the jQuery! :heart_eyes: Haven't seen that in a while..

On a more serious note, is listening for 'change' the expected behavior, or is 'blur' more user friendly? I'm no UX expert by any means.. Just thinking out loud here. I personally prefer 'on change'.

File: src/Surfnet/StepupRa/RaBundle/Resources/public/less/style.less [hunk 2]
```diff
@@ -233,9 +233,22 @@ form[name="select_institution"] {
...
+                text-align: left;
+            }
         }
     }
 }
+
+#select_institution_select_and_apply {
+    display: block;
+}
+.js #select_institution_select_and_apply {
```
Comment (MKodde) [inline src/Surfnet/StepupRa/RaBundle/Resources/public/less/style.less]: Nice touch to keep support for non-js believers!

File: src/Surfnet/StepupRa/RaBundle/Resources/public/js/stepup-ra.js (hunk 1 above)
Comment (pablothedude) [reply src/Surfnet/StepupRa/RaBundle/Resources/public/js/stepup-ra.js]: The `change` event is the one you'd like else if you get focus and you lose it your page is getting refreshed even if the value isn't changed at all.

File: src/Surfnet/StepupRa/RaBundle/Resources/public/less/style.less (hunk 2 above)
Comment (pablothedude) [reply src/Surfnet/StepupRa/RaBundle/Resources/public/less/style.less]: And alson the most easiest solution.

**Comment by pablothedude** [comment]
> The changes look good to me, on this branch I am not able to view the changes as I run into a fatal error:
> 
> ```
> Fatal Error: Call to undefined method Surfnet\StepupMiddlewareClientBundle\Identity\Dto\Profile::getRaaInstitutions()
> 
> {
>   /src/Stepup-RA/src/Surfnet/StepupRa/RaBundle/Controller/RaaController.php:46 {
>     › } else {
>     ›     $choices = $profile->getRaaInstitutions();
>     ›     $institution = reset($choices);
>   }
>   /src/Stepup-RA/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:151 {}
>   /src/Stepup-RA/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:151 {}
>   /src/Stepup-RA/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:68 {}
>   /src/Stepup-RA/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:200 {}
>   /src/Stepup-RA/web/app_dev.php:16 {}
>   /src/Stepup-RA/web/app_dev.php:0 {}
> }
> ```

I needed to update the client bundle which I did

**Pull request merged**